numpy/core/tests/test_simd_module.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+from numpy.core._simd import targets
+
+npyv = None
+npyv2 = None
+for target_name, npyv_mod in targets.items():
+    if npyv:
+        if npyv_mod and npyv_mod.simd:
+            npyv2 = npyv_mod
+            break
+        continue
+    if npyv_mod and npyv_mod.simd:
+        npyv = npyv_mod
```

Comment:
I'm confused by what's going on here. Is this equivalent to:
```suggestion
npvs = [npyv_mod for npyv_mod in targets.values() if npyv_mod and npyv_mod.simd]

# take only the first two because ???
npv, npv2 = (npvs + [None, None])[:2]

Reply:
yes it's equlivent to your suggetion.

> take only the first two because ???

This testing unit only testing the sanity of converstion and data functionalty that common used by all submodules.
therfor all we need is just to take one submodule that represents any of enabled SIMD extentions to run the test on it
and the second submodule required to run only one check related to the possiblity of mixing the data types among each submodule.

Reply:
Thanks, that makes sense. Could you add a comment to that effect?